site/content/recipes/find_a_reviewer.md

```diff
@@ -0,0 +1,160 @@
+# Objective
+
+Upon completing this recipe, you as the _code author_ will have found someone
+suitable to perform the role of _reviewer_ in the code review.
+
+# Steps
+
+1. Identify the top areas to search for candidate reviewers.
+  - Your research group
+  - Research Software Engineer (RSE) group at your institution
+  - Language or library specific groups at your institution
+  - Open source or research communities you contribute to
+  - Others?
+  - Last resort is to throw it up on Code Review Stack Exchange
+    <https://codereview.stackexchange.com/>
+2. Pre-screen candidates based on your existing knowledge and some criteria
+   (below).
+3. Contact candidates directly or make advertisement.
+4. Repeat 1 to 3 until you get a bite.
+5. Confirm candidate can fulfill minimum thresholds.
+
+# Description
+
+It might seem strange, but code review has something in common with partner
+dancing: it takes two to tango and (at least) two to review code. Without a
+reviewer, the code review cannot happen. And just as a good dance partner is
+essential for a well-performed routine, a suitable reviewer ensures that the
+code review will be a rewarding and valuable experience. Finding someone to
+fill this important role can be one of the more difficult steps in getting a
+code review off the ground — or dance floor, depending on how far you want the
+analogy to go.
+
+Perhaps the main barrier to finding a reviewer is misconceptions about what
+experience a reviewer should possess. As a lone coder, you might be thinking,
+"Only software developers do code review, so I should probably find someone
+outside of research." Or "The reviewer definitely needs to be more experienced
+than me." We want to be absolutely clear on this point: almost _any_ researcher
+who codes can be a good reviewer of your research code.
```

Comment:
I like that! This is a crucial point

Reply:
The implication of the way this is written is that a researcher who codes is not a software developer and I don't think that's the best way to put it. The two things are not mutually exclusive. This way of thinking does not help the researchers who code who often already feel they're less than the "true" software developers. While the message is good, I think the way it was written does not helps to break that division.

Reply:
> a researcher who codes is not a software developer

I also think that this is an unfortunate misconception and that we should do our best to counter it. Not sure we have consensus on that though.

Maybe making this paragraph a bit shorter would help?

```
The good news is, almost any researcher who code is a good candidate for being a reviewer.
Particularly, the belief that only software development experts should review code is an unfortunate 
misconception.  That being said, ...
```

Reply:
As mentioned in our previous meeting, it certainly wasn't my intention to imply a divide between researchers who code and software developers. I think it is a nuanced question, and there is a spectrum. In any case, I wholeheartedly disagree with looking at researchers who code as second class compared to RSEs or commercial software developers. I have reworded with Thibault's suggestion in 9f87163 to avoid any possible negative interpretations.